setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+
+from rna_cd import VERSION
+
+setup(
+    name="rna_cd",
+    description="RNA contamination detection",
+    version=str(VERSION),
+    author="Sander Bollen",
+    author_email="a.h.b.bollen@lumc.nl",
+    license="AGPLv3",
+    packages=find_packages(),
+    install_requires=[
```

Comment:
I miss `python_requires`. Could you also clarify why it needs to be 3.6 or higher with a comment? (I.e why not also 3.5?).

Reply:
Basically because my dev machine has version 3.6 as the default python3 :see_no_evil: